.github/workflows/clang-tidy.yml

```
@@ -0,0 +1,31 @@
+name: clang-tidy
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: alexays/waybar:debian
+    steps:
+      - uses: actions/checkout@v3
+      - name: configure
+        run: |
+          meson -Dcpp_std=c++20 build  # necessary to generate compile_commands.json
+          ninja -C build               # necessary to find certain .h files (xdg, wayland, etc.)
+      - uses: zjeffer/cpp-linter-action@test/process-compilation-database
```

The dev branch that this fork uses no longer exists. The fixes it introduced are now part of the v2.7.5 release; please update this line:
```suggestion
      - uses: cpp-linter/cpp-linter-action@v2
```